Add empty slides between topics

JakobMiksch opened the pull request:
- Add empty slides between topics

- we COULD also add black slides or some other kind of separator

| ![image](https://user-images.githubusercontent.com/15704517/143427764-61985ae5-dd61-4b27-8e33-2a3da38ca744.png) | ![image](https://user-images.githubusercontent.com/15704517/143442201-bd04696c-a955-490a-8ccc-7d3ec00b1be5.png)  | ![image](https://user-images.githubusercontent.com/15704517/143427801-fb298dd0-c11e-4690-a33a-16a03c9b8fd3.png) 
|---|---|---|

JakobMiksch committed:
Add empty slides between topics

chrismayer commented:
If you have a good idea for a good type of separator I would prefer this. Vertical bar, or an icon? Coloring the empty slide differently than the other slides is also not the worst idea. Maybe some grayish color, so it looks like disabled? Please go a pragmatic way without too much effort here.

JakobMiksch committed:
Add IDs to start of topics

JakobMiksch committed:
Add background color for empty slides

JakobMiksch commented:
@chrismayer 

- I added a gray background - Reveal also makes a good transition from the standard background color to the gray one.
- additionally I added section titles - this is useful for referencing them via link. e.g. `www.example.com/02-Technologie#/schnittstellen`

chrismayer reviewed (approved):
LGTM, please merge.
